pkg/sdk/api/v1beta1/fluentbit_types.go

```diff
@@ -38,6 +38,7 @@ type FluentbitSpec struct {
 	PositionDBLegacy     *KubernetesStorage          `json:"position_db,omitempty"` // deprecated, use PositionDB instead
 	PositionDB           KubernetesStorage           `json:"positiondb,omitempty"`
 	MountPath            string                      `json:"mountPath,omitempty"`
+	ExtraVolumeMount     []VolumeMount               `json:"extraVolumeMount,omitempty"`
```

Comment:
Would you mind renaming this to `ExtraVolumeMounts`?

Reply:
ok right, done